.eslintignore

```diff
@@ -1 +1,3 @@
 lib
```

Comment:
That will no longer be used, right?

Reply:
yeah, can be removed. Thanks! 

Although I didn't remove `lib` from `.gitignore` so that previous builds are not accidentally committed. Let me know if you want it removed as well.

Reply:
Let's remove it. I expect developers to be responsible with what they commit, and we have a review process.

Reply:
Sure!